src/parser.rs

```diff
@@ -43,9 +45,7 @@ fn is_header_token_char(chr: u8) -> bool {
         | b'{'
         | b'}'
         | b' '
-        | b'\\' => false,
-        _ => true,
-    }
+        | b'\\')
```

Comment:
Are these changes purely stylistic or is there a functional change here? If it's stylistic only could you revert this?

Reply:
I only did this because I got this clippy lint and I implemented the suggested improvement: https://rust-lang.github.io/rust-clippy/master/index.html#match_like_matches_macro